scripts/index_aster_lpdaac.py

```diff
@@ -0,0 +1,640 @@
+                        --product aster_l1t_vnir /g/data/v10/ASTER_AU/2018.01.01
+    ./index_nci_aster_lpdaac.py create-vrt
+                        --product aster_l1t_vnir /g/data/v10/ASTER_AU/2018.01.01
+    ./index_nci_aster_lpdaac.py index-data
+                        --product aster_l1t_vnir /g/data/v10/ASTER_AU/2018.01.01
+
+
+::
+
+    psql -h agdcdev-db.nci.org.au
+    CREATE DATABASE aster_lpdaac WITH OWNER agdc_admin;
+    GRANT TEMPORARY, CONNECT ON DATABASE aster_lpdaac to public;
+
+aster_lpdaac.conf::
+
+    [datacube]
+    db_hostname: agdcdev-db.nci.org.au
+    db_port: 6432
+    db_database: aster_lpdaac
+
+::
+
+    datacube --config aster_lpdaac.conf system init
+
+::
+
+    for i in /g/data/v10/ASTER_AU/*; do
+        ./index_nci_aster_lpdaac.py --config aster_lpdacc.conf index-data
+                                    --product aster_l1t_vnir $i
+    done
+
+"""
+import json
+import logging
+import uuid
+from datetime import datetime, timezone
+from pathlib import Path
+import functools
+import yaml
+
+import click
+import numpy as np
+from osgeo import gdal, osr
+import rasterio
+
+from datacube import Datacube
+from datacube.index.hl import Doc2Dataset
+
+
+LOG = logging.getLogger(__name__)
+
+PRODUCTS = {'aster_l1t_vnir': {'ImageData2', 'ImageData1', 'ImageData3N'},
+            'aster_l1t_swir': {'ImageData4', 'ImageData5', 'ImageData6', 'ImageData7', 'ImageData8', 'ImageData9'},
+            'aster_l1t_tir': {'ImageData10', 'ImageData11', 'ImageData12', 'ImageData13', 'ImageData14'}}
+
+EXTRA_METADATA_PREFIXES = {
+    'aster_l1t_vnir': {'include_only': {'ASTER', 'CORRECT', 'EAST', 'FLY', 'GAIN', 'INPUT', 'LOWER', 'MAP',
+                                        'NORTH', 'NUMBERGCP', 'ORBIT', 'POINT', 'QAPERCENT', 'RECURRENT', 'SCENE',
+                                        'SIZE', 'SOLAR', 'SOUTH', 'UPPER', 'UTMZONENUMBER', 'WEST'}},
+    'aster_l1t_swir': {'include_only': {'ASTER', 'CORRECT', 'EAST', 'FLY', 'GAIN', 'INPUT', 'LOWER', 'MAP',
+                                        'NORTH', 'NUMBERGCP', 'ORBIT', 'POINT', 'QAPERCENT', 'RECURRENT', 'SCENE',
+                                        'SIZE', 'SOLAR', 'SOUTH', 'UPPER', 'UTMZONENUMBER', 'WEST'}},
+    'aster_l1t_tir': {'exclude': {'BAND', 'CALENDAR', 'COARSE', 'FUTURE', 'GEO', 'HDF', 'IDENT', 'IMAGE',
+                                  'PGE', 'PROCESSED', 'PROCESSING', 'RADIO', 'RECEIVING', 'REPROCESSING', 'SOURCE',
+                                  'TIME', 'UTMZONECODE'}}
+}
+
+
+@click.group(help=__doc__)
+@click.option('--config', '-c', help="Pass the configuration file to access the database",
+              type=click.Path(exists=True))
+@click.pass_context
+def cli(ctx, config):
+    """ Used to pass the datacube index to functions via click."""
+    ctx.obj = Datacube(config=config).index
+
+
+@cli.command()
+@click.argument('path')
+@click.option('--product', help='Which ASTER product? vnir, swir, or tir')
+def create_vrt(path, product):
```

Comment:
Where are the VRTs written? This isn't mentioned anywhere I can see. Are they written into the same directory as the source HDFs?

Reply:
Yes written onto the same source HDFs (including dataset YML files) and updated implementation details doc section accordingly